config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  namespace :api, defaults: { format: 'json' } do
+    namespace :v1, defaults: { format: 'json' } do
+      get 'messages', to: 'greeting#index'
```

Comment:
- [ ] I was getting an exception error when I visited this route to get the message. This is because there is not Greeting controller, what you have is a Greetings controller. You are missing an "s" in `'greeting#index'`. It should be this: `'greetings#index'`

  ![no-cont](https://user-images.githubusercontent.com/81468675/183674756-66a04b65-0ffa-4847-8202-f930fde8cbfb.jpg)